app/assets/javascripts/discourse/app/controllers/user-invited-show.js

```diff
@@ -31,10 +31,16 @@ export default Controller.extend({
       this.user,
       this.filter,
       this.searchTerm
-    ).then(invites => this.set("model", invites));
+    ).then((invites) => this.set("model", invites));
   }, INPUT_DELAY),
 
   inviteRedeemed: equal("filter", "redeemed"),
+  invitePending: equal("filter", "pending"),
+
+  @discourseComputed("filter")
+  inviteLinks(filter) {
+    return filter === "links" && this.currentUser.get("admin");
```

Comment:
do we need the get here ? if currentUser is null having a get won't save you from an error

Reply:
Good catch! Fixed.